<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SamplePlugin</class>
 <widget class="QDockWidget" name="SamplePlugin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DockWidget</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Select marker</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="_drop_markerselect">
        <item>
         <property name="text">
          <string>Markerpose</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Marker1.ppm</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Marker2a.ppm</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Marker2b.ppm</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Marker3.ppm</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Select sequence</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="_drop_sequenceselect">
        <item>
         <property name="text">
          <string>MarkerMotionSlow.txt</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MarkerMotionMedium.txt</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MarkerMotionFast.txt</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Background</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="_spinBoxBackground">
        <item>
         <property name="text">
          <string>color1.ppm</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>color2.ppm</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>color3.ppm</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>lines1.ppm</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>texture1.ppm</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>texture2.ppm</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>texture3.ppm</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="_btnStart">
        <property name="text">
         <string>Start sequence</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="_btnStop">
        <property name="text">
         <string>Stop sequence</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="_btnRestart">
        <property name="text">
         <string>Restart sequence</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="_btnNextFrame">
        <property name="text">
         <string>Next frame</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>dt</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="_dtslider">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="pageStep">
         <number>5</number>
        </property>
        <property name="value">
         <number>1000</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Go to frame in sequence</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="_spinBox">
        <property name="maximum">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="_slider">
        <property name="maximum">
         <number>500</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="_label">
        <property name="text">
         <string>Label</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
